### PR TITLE
Add healthcheck endpoint

### PR DESCRIPTION
**Pull request by CatalinVoineag**
## Context

Add a simple healthcheck page so kubernetes can monitor the container.

## Changes proposed in this pull request

New healthcheck route

## Guidance to review

Just visit `/healthcheck`

- 2 file changes are because of the linter.

## Link to Trello card

https://trello.com/c/z21wL0OH/77-add-healthcheck-endpoint

## Things to check

- [ ] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [ ] API release notes have been updated if necessary
- [ ] If it adds a significant user-facing change, is it documented in the [CHANGELOG](CHANGELOG.md)?
- [ ] Required environment variables have been updated or [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

## Screenshots

Screenshot from 2023-12-18 13-58-41

### CODE REVIEW

**Review by elceebee — APPROVED** [review]

File: config/routes.rb [hunk 1]
```diff
@@ -16,4 +16,6 @@
 
   draw :placements
   draw :claims
+
+  get "/healthcheck", to: proc { [200, {}, ["OK"]] }
```
Comment (Nitemaeric) [inline config/routes.rb:20]: Maybe https://edgeapi.rubyonrails.org/classes/Rails/HealthController.html could be used?

```suggestion
  get "/healthcheck", to: "health#show"
```

Rails supposedly defaults this to `/up`, but locally it isn't in there for me.

**Review by Nitemaeric — COMMENTED** [review]
Nitpick, but could you split out the lint changes into a separate commit please?

File: config/routes.rb (hunk 1 above)
Comment (CatalinVoineag) [reply config/routes.rb]: I've started reading that and figure out that we don't check if the DB is up or if we add sideqkiq and redis later on, we don't check the health of those. Refactored this into a `Heartbeat` controller, similar to the one in register. To check for these things

**Review by JamieCleare2525 — APPROVED** [review]

File: config/routes.rb [hunk 2]
```diff
@@ -16,4 +16,6 @@
 
   draw :placements
   draw :claims
+
+  get :healthcheck, controller: :heartbeat
```
Comment (Nitemaeric) [inline config/routes.rb:20]: Logically, this is all good 👍 Out of curiousity, why is the controller "Heartbeat" and the action "healthcheck"? Are you planning to add additional heartbeats?

Comment (CatalinVoineag) [reply config/routes.rb:20]: I don't like having methods that are named the same as the class they are in, and this way we can add additional heartbeats if we need to

**Pull request merged**